linux/step04_all_omero_install.sh

```diff
@@ -8,9 +8,11 @@ set -eux
 
 #start-install
 if [ $OMEROVER == "latest" ]; then
+	# version number and ice version to be dropped
```

Comment:
yes, who and when will drop them ?

Reply:
Next release if https://github.com/ome/openmicroscopy/pull/6414 is accepted and included